tendermint/tests/support/lite/header_tests_failing.json

```diff
@@ -0,0 +1,169 @@
+            "num_txs": "0",
+            "total_txs": "0",
+            "last_block_id": {
+              "hash": "",
+              "parts": {
+                "total": "0",
+                "hash": ""
+              }
+            },
+            "last_commit_hash": "",
+            "data_hash": "",
+            "validators_hash": "75B9F27F25F3515EF7CA61DD42C272CCC280C537E6A9ED493710EB361CB90B0D",
+            "next_validators_hash": "75B9F27F25F3515EF7CA61DD42C272CCC280C537E6A9ED493710EB361CB90B0D",
+            "consensus_hash": "048091BC7DDC283F77BFBF91D73C44DA58C3DF8A9CBC867405D8B7F3DAADA22F",
+            "app_hash": "41CAFAE31CC70F5801FA1016A2DD54A9BCB8201B5B389919FE9976762532C516",
+            "last_results_hash": "",
+            "evidence_hash": "",
+            "proposer_address": "01F527D77D3FFCC4FCFF2DDC2952EEA5414F2A33"
+          },
+          "commit": {
+            "block_id": {
+              "hash": "28B21F5C724BC1FF9EB32686F69CE150350F990A491E58DE8BF65A72C7DD491C",
+              "parts": {
+                "total": "1",
+                "hash": "57085A01842E9E8C7461F377B074CCA52D0D43BCDB2B82FFAE4EB5A0DFB779AE"
+              }
+            },
+            "precommits": [
+              {
+                "type": 2,
+                "height": "1",
+                "round": "1",
+                "block_id": {
+                  "hash": "28B21F5C724BC1FF9EB32686F69CE150350F990A491E58DE8BF65A72C7DD491C",
+                  "parts": {
+                    "total": "1",
+                    "hash": "57085A01842E9E8C7461F377B074CCA52D0D43BCDB2B82FFAE4EB5A0DFB779AE"
+                  }
+                },
+                "timestamp": "2019-11-02T15:04:10Z",
+                "validator_address": "01F527D77D3FFCC4FCFF2DDC2952EEA5414F2A33",
+                "validator_index": "0",
+                "signature": "y6Yd/g9hBzatGEm8WCVXgOks5rmnOlg2fsin2comsb7Y2oVM0/2sCucSPapci7VXSrUNJHC3ruE7onIylrFUDA=="
+              }
+            ]
+          }
+        },
+        "next_validator_set": {
+          "validators": [
+            {
+              "address": "01F527D77D3FFCC4FCFF2DDC2952EEA5414F2A33",
+              "pub_key": {
+                "type": "tendermint/PubKeyEd25519",
+                "value": "OAaNq3DX/15fGJP2MI6bujt1GRpvjwrqIevChirJsbc="
+              },
+              "voting_power": "50",
+              "proposer_priority": "0"
+            }
+          ],
+          "proposer": {
+            "address": "01F527D77D3FFCC4FCFF2DDC2952EEA5414F2A33",
+            "pub_key": {
+              "type": "tendermint/PubKeyEd25519",
+              "value": "OAaNq3DX/15fGJP2MI6bujt1GRpvjwrqIevChirJsbc="
+            },
+            "voting_power": "50",
+            "proposer_priority": "0"
+          }
+        },
+        "trusting_period": "10800000000000",
+        "now": "2019-11-02T15:05:00Z"
+      },
+      "input": [
+        {
+          "signed_header": {
+            "header": null,
```

Comment:
Currently, this (null `header`) won't even parse correctly. (In the SignedHeader struct a `header` isn't optional).